package.json

```diff
@@ -30,6 +30,10 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
```

Comment:
Might have to update the typescript dependency to `^4.1.3`. See: https://github.com/chakra-ui/chakra-ui/discussions/3686#discussioncomment-550611

Without the change, two typescript errors occur when running `npm run build`